### ISSUE TITLE
How can we stop seeking feature of exoplayer when it is not required?

### ISSUE THREAD

**Issue by vyagrah** [google/ExoPlayer#10099]
In my app I have no use of seeking feature provided by exoplayer library.
(not using seekTo(), seeBack(), seekForward(), seekToNext(), ... meathods)

Is there any way to stop seeking?
Does doing that improve performance of our player?

**Comment by icbaker** [comment]
The obvious answer seems to be: just don't call those methods?

I assume this is related to https://github.com/google/ExoPlayer/issues/10081 but I'm not quite clear on the exact details.

Could you please provide complete information to enable us to understand your question. We have a template for questions: https://github.com/google/ExoPlayer/blob/release-v2/.github/ISSUE_TEMPLATE/question.md

**Comment by vyagrah** [comment]
That is true. It is related to #10081 . I don't understand which information you needed more.

As I said at #10081 , I noticed when I play normal stream `onTimelineChanged `called only one time, but with HLS streams it is called again and agian. I also noticed that exo_position gives same value as `player.getDuration()`.
(I am playing radio streams, some of which are HLS and some are NON HLS.)

I thought if I stop this **seeking** then `onTimelineChanged `not get called again and again on HLS streams and I get right played duration as I already getting on non HLS streams using `player.getDuration()` or better by using `exo-position` in xml. I just want to show the time user played stream till now. Working fine with non HLS streams, giving negative values on HLS Streams

I am new with exoplayer, HLS, m3u8, ICY stuff. I am trying to learn.
